test/cxx/Core/ApplicationPool/PoolTest.cpp

```diff
@@ -2026,4 +2026,29 @@ namespace tut {
 
 
 	/*****************************/
+
```

Comment:
You should move this code to before the `/*****/` marker. Code after the marker is meant for the Passenger Enterprise codebase.